unfetter-discover/unfetter
# Issue 889 (sample object assessments)

Fixes unfetter-discover/unfetter#889

# To test:

1. Pull issue-889 branch
2. Restart unfetter-discover-processor
3. Verify there are 2 object assessments in the stix collection
4. You can use these DB commands to verify in Robo 3T:

`db.getCollection('stix').find({"stix.type" : "x-unfetter-object-assessment"})`
